vowpalwabbit/slates.cc

```diff
@@ -70,16 +70,18 @@ void slates_data::learn_or_predict(VW::LEARNER::multi_learner& base, multi_ex& e
     else if (slates_label.type == slates::example_type::slot)
     {
       ccb_label.type = CCB::example_type::slot;
+      ccb_label.explicit_included_actions = v_init<uint32_t>();
+      for (const auto index : slot_action_pools[slot_index])
+      {
+        ccb_label.explicit_included_actions.push_back(index);
+      }
+
       if (global_cost_found)
       {
         ccb_label.outcome = new CCB::conditional_contextual_bandit_outcome();
         ccb_label.outcome->cost = global_cost;
         ccb_label.outcome->probabilities = v_init<ACTION_SCORE::action_score>();
```

Comment:
what about this line? how does that relate to global_cost_found

Reply:
Probabilities are associated with the global cost